src/pydata_sphinx_theme/assets/styles/extensions/_sphinx_design.scss

```diff
@@ -66,25 +66,40 @@ $all-colors: map.merge($pst-semantic-colors, $extra-semantic-colors);
 
 @mixin create-sd-colors($value, $name) {
   // define the pst variables, so that downstream user overrides will work
+  @debug "Creating color variables for semantic color: #{$name}";
```

Comment:
It looks like we're able to create AA conforming colors for everything except warning-highlight in light mode: 

| warning-highlight | warning-highlight-text |
| --- | --- |
| `#d27a07` | `#14181e` | 

Contrast ratio 4.4:1.

Reply:
Yeah there is no WCAG conformant combination with our text colours and the orange (at least how this is being calculated).
The alternative would be to manually specify all the colours `pst` and `sd` and use our colour palette. 
I mean we can do that since the colours are always present like mentioned in the comment above. 
But there are tradeoffs to each approach.

Reply:
That makes sense. I was actually trying to say that I was impressed that there was only one semantic color that didn't work. And it's not that far from the 4.5 target.